store/datastore/users.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/drone/drone/model"
 	"github.com/russross/meddler"
+	"sort"
 )
```

Comment:
imports should be grouped and sorted based on Go style guide. See https://github.com/golang/go/wiki/CodeReviewComments#imports